reference/2_constructing_data/2_3_merge_concat/README.md

```diff
@@ -0,0 +1,104 @@
+# Merging & concatenation
+
+## Example - Merging dictionaries
+
+> **_Try out:_**
+>
+> Place node **tutorials/data_constr/merge_dict** or **[download .cranqj](cranqj/data_constr_merge_dict.cranqj)** 
+
+Dictionaries in Cranq can be merged with the ```data/dictionary/Merger``` node.
+- The node simply takes 2 dictionaries
+- If the keys conlict, then the values in "b" take precedence
+
+![](images/2021-07-19-14-23-42.png)
+
+### Sample output:
+
+```json
+{
+  "a": "b",
+  "c": 3,
+  "d": "e"
+}
+```
+
+## Example - Concatenating arrays
+
+> **_Try out:_**
+>
+> Place node **tutorials/data_man/concat_array** or **[download .cranqj](cranqj/data_constr_concat_array.cranqj)** 
+
+Concatenating arrays in Cranq can be performed with the ```data/array/Concatenator``` node.
+
+
+![](images/2021-07-19-14-27-25.png)
+
+- It simply takes 2 arrays & applies the items of "b" onto "a", in order
+- The example combines the node (```tutorials/data_constr/syncer_array_emps```) created in **[Example - Initializing an array with values](../2_4_syncer_splitter/README.md)** & the following test data:

```

Comment:
We refer here to an example that comes later in the flow of reading.
May be better to refer to the example "Building a complex type" https://github.com/Cranq-io/cranq-tutorials/blob/feature/reference/reference/2_constructing_data/2_2_builders/README.md#example---building-a-complex-type

Reply:
Perhaps the order can be flipped between them (2.3 syncer -> 2.4 merger), I will see if it makes sense.